frontend_new/src/App.tsx

```diff
@@ -23,9 +23,35 @@ function App() {
       setThemeChoice(dark);
     }
   };
+  const [bookmarks, setBookmarks] = React.useState({});
+  function handlebookmark(
+    web_link: string,
+    primary: string,
+    secondary: string,
+    id: integer
+  ) {
+    if (id in bookmarks) {
+      var temp_bookmarks = bookmarks;
```

Comment:
**[Suggest]** Vaguely remember using **let** is better than using **var**

Reply:
Changed in latest commit!